config/jobs/kubernetes/sig-security/cvelist-public.yaml

```diff
@@ -0,0 +1,26 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: validate-cve-files
+    decorate: true
+    always_run: false
+    optional: true
+    run_if_changed: 'CVE.*json$'
+    spec:
+      containers:
+      - image: golang
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |
+          set -euo; \
```

Comment:
doesn't `set -o` need an argument? I think this should be `set -euo pipefail`, but I could be wrong...

Reply:
fixed.